inkpot/filter.py

```diff
@@ -12,7 +12,7 @@ def filter_file(file_path):
     with open(file_path, "r") as file:
         for cnt, line in enumerate(file):
             # print("Line {}: {}".format(cnt, line))
-            if "def" in line:
+            if line.lstrip().find("def ") == 0:
```

Comment:
Wont `find("def "` be problem? 
because you are now looking for `def (some_arg)` instead of `def(some_arg)`
The space migt be a problem.

Reply:
I don't think so becuase functions in python are declared like: 
```python
def func_name(parameter):
    pass
```
We are then looking for "def " with a space to make it more robust.

Reply:
But it should not really matter either way.